matrix/main.py

```diff
@@ -44,7 +45,10 @@ def setup(matrix, args=None):
     parser.add_argument("-i", "--interval", default=5.0, type=float)
     parser.add_argument("-p", "--path", default=Path.cwd() / "tests",
                         type=Path)
-    parser.add_argument("config_file", type=open)
+    parser.add_argument("-c", "--config-file",
+                        default=resource_filename(__name__, 'matrix.yaml'))
```

Comment:
This also touches on the discussion we had about whether we wanted BT to be the thing that discovers and "combines" (in some way) the in-bundle test with the standard / default case, and my understanding was that we didn't want that.

If we do, we could rip out the bundle fetch code entirely and always assume we're deploying a local bundle, with BT being responsible for fetching it and running Matrix from within the bundle directory to ensure the right paths.